.prettierrc

```diff
@@ -1,4 +1,4 @@
 {
 	"trailingComma": "es5",
-	"singleQuote": true
-}
\ No newline at end of file
+	"singleQuote": false
```

Comment:
So, unfortunately, just doing this will not replace all existing single quotes into double quotes. It will just prevent future ones :P, but it's a necessary step going forward haha so thanks! If you want to also replace the existing single quotes while you're at it, that would be great.

Additionally, since double quotes are the prettier default, just removing the line completely will have the same effect 👍